cachi2/interface/cli.py

```diff
@@ -165,18 +165,12 @@ def fetch_deps(
     gomod_vendor: bool = typer.Option(
         False,
         "--gomod-vendor",
-        help=(
-            "Fetch go deps via 'go mod vendor' rather than 'go mod download'. If you "
-            "have a vendor/ dir, one of --gomod-vendor/--gomod-vendor-check is required."
-        ),
+        help=("Deprecated flag. Will have no effect, and its use is discouraged."),
```

Comment:
I think we want to retain the original meaning of the flag as well, but then make 'deprecated' in capitals.

Reply:
Ok, I changed it. LMK if you are ok with it.